arangod/Transaction/Context.cpp

```diff
@@ -109,10 +108,7 @@ void transaction::Context::cleanup() noexcept {
 
   _stringBuffer.reset();
 
-  if (_ownsResolver) {
-    delete _resolver;
-    _resolver = nullptr;
-  }
```

Comment:
Awesome. I remember we once had functionality to pass an existing resolver into a transaction::Context, but that seems to have been removed a while ago already. So going with a unique_ptr is exactly the right thing here :+1:

Reply:
I figured there had to be a historical reason for the code to look like that. 😉